app/js/views/themes-list-item.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'components/react';
+
+export default class ThemesListItem extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      enabled: props.theme.enabled
+    };
+
+    this.foxbox = props.foxbox;
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleOnDelete = this.handleOnDelete.bind(this);
+  }
+
+  /**
+   * Activate or deactivate a recipe.
+   *
+   * @param {SyntheticEvent} evt
+   */
+  handleOnChange(evt) {
+    const enabled = evt.target.checked;
+
+    this.setState({ enabled }); // Optimistic update.
+
+    this.foxbox.recipes.toggle(this.props.theme.id, enabled)
+      .catch(error => {
+        this.setState({ enabled: !enabled }); // Revert back to previous value.
+        console.error(error);
+      });
+  }
+
+  /**
+   * Delete a recipe.
+   */
+  handleOnDelete() {
+    this.foxbox.recipes.remove(this.props.theme.id)
+      .then(() => {
+        this.props.update();
+      })
+      .catch(console.error.bind(console));
```

Comment:
I like the `bind` pattern in general because it will display all the arguments if there are more than one.